manifests/init.pp

```diff
@@ -88,7 +88,7 @@
     'force_rebuild'     => 'UNSET',
   }
 
-  $config = merge($config_defaults, $config_overrides)
+  $config = stdlib::merge($config_defaults, $config_overrides)
```

Comment:
This will cause this module to required stdlib 9.x which I'd like to avoid for now.  I think this can be replaced with:

```
$config = $config_defaults + $config_overrides
```

Reply:
Understandable, I replaced the `merge` line like you suggested so it doesn't require stdlib.